app/src/main/res/values/strings.xml

```diff
@@ -25,12 +25,12 @@
     <string name="home_notice_content">Download Magisk ONLY from the official GitHub page. Files from unknown sources can be malicious!</string>
     <string name="home_support_title">Support Us</string>
     <string name="home_item_source">Source</string>
-    <string name="home_support_content">Magisk is, and always will be, free and open-source. You can however show us that you care by sending a small donation.</string>
+    <string name="home_support_content">Magisk will always be copylefted libre software.\nUse,study, change and share; with all.\nShow you care by sending a donation.</string>
     <string name="home_installed_version">Installed</string>
     <string name="home_latest_version">Latest</string>
     <string name="invalid_update_channel">Invalid Update Channel</string>
     <string name="uninstall_magisk_title">Uninstall Magisk</string>
-    <string name="uninstall_magisk_msg">All modules will be disabled/removed!\nRoot will be removed!\nYour data potentially encrypted if not already!</string>
+    <string name="uninstall_magisk_msg">All root privileges will be removed\nAll Magisk modules will be turned of and removed!\nStorages decrypted in Magisk will be re-encrypted!</string>
```

Comment:
```suggestion
    <string name="uninstall_magisk_msg">All root privileges will be removed\nAll Magisk modules will be disabled and then removed!\nStorage devices decrypted through the use of Magisk will be re-encrypted!</string>
```

Reply:
Maybe just "will be removed" would be more clear. I was trying to remember if that actually happens when using that option or if they technically just stay in in /data/adb/modules

Reply:
That it is ultimately removed isn't incorrect, and shorter is better. Better to add it back in once known.

Reply:
Complete uninstall does remove all modules, so let's keep it slim and go with:
```
Root will be removed!\nAll modules will be removed!\nAny storage device decrypted through the use of Magisk will be re-encrypted!
```
🙌

Reply:
They could be removed, but still run from memory. Going with the shorter option.

Reply:
What the fuck are you talking about? It removes them and then reboots the device. I'm increasingly seeing you have no idea what the strings you are trying to "improve" actually mean nor what the features they're attached to do.

Reply:
If you uninstall (remove) a program from a Linux system, it will continue to run ephemerally till the session ends if open. Unloading modules on the fly is not something Linux kernel devs support or test for, so one could carry this over. Least the understanding could be that, however ill devised it is. By removing "disabled" the user isn't specifically in the know.
I went with your shorter option.